v0/pipeline.json

```diff
@@ -61151,7 +61151,7 @@
             } ],
             "$schema" : "http://json-schema.org/draft-07/schema#",
             "type" : "object",
-            "required" : [ "deploymentSlot", "webApp" ],
+            "required" : [ "deploymentSlot", "functionApp" ],
```

Comment:
is this a new feature? no customers will have it? since this is a required field.

Reply:
yes, we found out in testing on QA.